config/state.ts

```diff
@@ -1,15 +1,7 @@
 import en1000 from '../wordlists/en1000.json';
 
 type Event =
-	| 'disableSFXConfetti'
-	| 'enableSFXConfetti'
-	| 'failureSlow'
-	| 'failureTypo'
-	| 'gameComplete'
-	| 'loadState'
-	| 'streakComplete'
-	| 'type'
-	| 'wordComplete';
+	'disableSFXConfetti' | 'disableSound' | 'enableSFXConfetti' | 'enableSound' | 'failureSlow' | 'failureTypo' | 'gameComplete' | 'loadState' | 'streakComplete' | 'type' | 'wordComplete';
```

Comment:
Please restore this union-type list to its original one-entry-per-line format, as it's becoming quite a lengthy declaration.